scripts/fuse/elektra_fuse/elektra_fuse_interface.py

```diff
@@ -0,0 +1,218 @@
+#stdlib imports
+import errno, stat, time, re, psutil, logging, subprocess, os
+from pathlib import Path
+
+#fuse imports
+from fuse import FUSE, FuseOSError, Operations, LoggingMixIn
+
+#project imports
+from elektra_util import *
+import mock_context
+
+startup_time = time.time()
+
+#TODO: give appropriate error codes (instead of readonly fs) when non-authenticated user tries to do illegal things
+
+def getattr(path, fh=None):
+
+    is_value_of_file = Path(path).name == dir_file_special_name
+
+    is_dir, is_file = key_type(path)
+
+    if is_value_of_file:
+        mode = stat.S_IFREG
+        #resolve the liml (/.../@elektra.value) key to the real key (the parent)
+        path = str(Path(path).parent)
+    elif not is_dir and not is_file:
+        mode = errno.ENOENT
+    elif is_dir and is_file:
+        mode = stat.S_IFDIR
+    elif is_dir:
+        mode = stat.S_IFDIR
+    elif is_file and has_meta(path, "meta:/fuse-directory"):
+        mode = stat.S_IFDIR
+    elif is_file:
+        mode = stat.S_IFREG
+
+    if mode == stat.S_IFREG:
+        try:
+            filesize = size_of_file(path) 
+        except KeyError: #key does not exist
+            mode = errno.ENOENT
+
+
+    try:
+        kdb_file_stat = _stat_kdb_file(path)
+        st_mode_only_permission_bits = kdb_file_stat.st_mode & 0o777
+    except FileNotFoundError:
+        # some keys, e.g. "system:/elektra" resolve to a file like "/etc/kdb/elektra.ecf", which does not exist
+        # for the filesystem to remain useful, in this case dummy file attributes are used
+        st_mode_only_permission_bits = 0o770
```

Comment:
```suggestion
        st_mode_only_permission_bits = 0o000
```

Maybe this makes clear that the file actually does not exist.